bitnami/keycloak/templates/statefulset.yaml

```diff
@@ -59,9 +59,43 @@ spec:
       {{- if .Values.podSecurityContext.enabled }}
       securityContext: {{- omit .Values.podSecurityContext "enabled" | toYaml | nindent 8 }}
       {{- end }}
-      {{- if .Values.initContainers }}
+      {{- if or .Values.initContainers .Values.auth.tls.enabled }}
       initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.auth.tls.enabled }}
+        - name: init-certs
+          image: busybox
```

Comment:
As per bitnami's policy, we have to use images created and maintained by us in our charts. Could you modify the used image here to `bitnami/minideb`? We should also include the `securityContext` and `resources` parameters for the initContainer and have this as parameters in `values.yaml`. You could use [this other chart](https://github.com/bitnami/charts/blob/638fdadaa907a36b0304d738cb30aef366cc3255/bitnami/dokuwiki/templates/deployment.yaml#L56) as a reference.

Reply:
Got it. I figured as much just didn't know which image to use. Thanks for the reference.